src/lib.rs

```diff
@@ -85,6 +89,24 @@ pub fn update_blocks(
     blocks_clone
 }
 
+/// Generate default topics that is operator address resolved to indexer address
+/// and then its active on-chain allocations
+pub async fn default_topics(operator_address: String) -> Result<Vec<String>, Box<dyn Error>> {
```

Comment:
Awesome stuff! My only question is whether it'll be better for this to be named `topics`/`allocations`/`allocations_hashes`? Since it's not _default_ it's specific for this Radio

Reply:
since this helper lives within this radio, I thought it is reasonable to call it default, but will update to `allocations_hashes`